device/celestica/x86_64-cel_e1031-r0/sonic_platform/common.py

```diff
@@ -195,7 +196,8 @@ def write_txt_file(self, file_path, value):
         return True
 
     def is_host(self):
-        return os.system(self.HOST_CHK_CMD) == 0
+        command = split(self.HOST_CHK_CMD)
+        return subprocess.run(command).returncode == 0

```

Comment:
>subprocess.run [](http://example.com/codeflow?start=15&length=14)

For this special case, the safe way is
`subprocess.run(['docker'], stdout=None, stderr=None).returncode`

Reply:
The stdout/stderr redirect is not needed inside `HOST_CHK_CMD` string.

Reply:
Fixed.

Reply:
Suggest this way.
```
HOST_CHK_CMD = ['docker']
```
Benefit:
1. minimal code change
2. string array will be super safe and one less command line parsing.